src/exitmap.py

```diff
@@ -270,11 +274,20 @@ def select_exits(args, module):
         hosts = [(socket.gethostbyname(host), port) for
                  (host, port) in module.destinations]
 
-    # '-e' was used to specify a single exit relay.
-
     if args.exit:
+        # '-e' was used to specify a single exit relay.
+
         exit_relays = [args.exit]
         total = len(exit_relays)
+    elif args.exit_file:
+        # '-E' was used to specify a file containing exit relays
+
+        try:
+	   exit_relays = [line.strip() for line in open(args.exit_file)]
```

Comment:
This line starts with a tab. Can you please turn it into spaces? Apart from that, the patch looks good to me.